mmv1/third_party/terraform/tests/resource_compute_instance_test.go

```diff
@@ -4138,6 +4169,68 @@ resource "google_compute_instance" "foobar" {
 `, instance)
 }
 
+func testAccComputeInstance_advancedMachineFeatures(instance string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-10"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance" "foobar" {
+  name         = "%s"
+  machine_type = "n1-standard-2" // Nested Virt isn't supported on E2 and N2Ds https://cloud.google.com/compute/docs/instances/nested-virtualization/overview#restrictions and https://cloud.google.com/compute/docs/instances/disabling-smt#limitations
+  zone         = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = data.google_compute_image.my_image.self_link
+    }
+  }
+
+  network_interface {
+    network = "default"
+  }
+//   advanced_machine_features {
+// 	threads_per_core = 1
+// 	enable_nested_virtualization = true
+//   }
```

Comment:
This should probably just be different values right? Or are you wanting to test adding the block altogether?

Reply:
A test to see what adding the block does. I will remove it from the 1st test.